gov2/secretsmanager/common/update.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//snippet-sourcedescription:[Update a secret in the AWS Secrets Manager ]
+//snippet-keyword:[secretsmanager]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[10/27/2021]
+//snippet-sourceauthor:[gangwere]
```

Comment:
At a minimum, include a Purpose section, which describes the purpose of this file. It should be outside the snippet tags. This purpose is to help orient users who land here. You could use the //snippet-sourcedescription content?